janusgraph-hbase-parent/janusgraph-hbase-10/src/main/java/org/janusgraph/diskstorage/hbase/HConnection1_0.java

```diff
@@ -46,4 +51,17 @@ public void close() throws IOException
     {
         cnx.close();
     }
+
+    @Override
+    public NavigableMap<HRegionInfo, ServerName> getRegionLocations(String tableName)
+        throws IOException
+    {
+        List<HRegionLocation> locations =
+            this.cnx.getRegionLocator(TableName.valueOf(tableName)).getAllRegionLocations();
+        NavigableMap<HRegionInfo, ServerName> result = new TreeMap<HRegionInfo, ServerName>();
+        for (HRegionLocation hrl : locations) {
+            result.put(hrl.getRegionInfo(), hrl.getServerName());
+        }
+        return result;
```

Comment:
Done, but requires an awkward lambda in the middle to allow for returning a sorted map.